docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -11,85 +15,339 @@ Where specific query examples are given, they are commonly provided in pairs usi
 
 pivot()
 -------
-Todo
+Returns a set of nodes that share a specified property of the same type / valu as the original set of nodes.
+
+Optional parameters:
+
+* **Return limit:** specify the maximum number of nodes returned by the ``pivot()`` query.
+
+  * ``limit=`` (operational syntax)
+  * ``^`` (macro syntax)
+
+**Operator syntax:**
+
+.. parsed-literal::
+
+  **pivot(** [ *<srcprop>* **,** ] *<dstprop>* [ **, limit=** *<limit>* ] **)**
+
+**Macro syntax:**
+
+.. parsed-literal::
+
+  [ *<srcprop>* **^** *<limit>* ] **->** *<dstprop>*
+
+**Examples:**
+
+* Pivot from a set of domains (``inet:fqdn`` nodes) in the working set to the DNS A records for those domains:
+  ::
+    pivot( inet:fqdn, inet:dns:a:fqdn )
+    
+    pivot( inet:dns:a:fqdn )
+    
+    inet:fqdn  ->  inet:dns:a:fqdn
+    
+    -> inet:dns:a:fqdn
+
+* Pivot from a set of domains in the working set to the DNS A records for those domains, but limit the results to 10 nodes:
+  ::
+    pivot( inet:fqdn, inet:dns:a:fqdn, limit=10 )
+    
+    pivot( inet:dns:a:fqdn, limit=10 )
+    
+    inet:fqdn^10  ->  inet:dns:a:fqdn
```

Comment:
Like the above, I think this example is confusing because it is not going to limit the results to 10 nodes

Reply:
Fixed / removed example.